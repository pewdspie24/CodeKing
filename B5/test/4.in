3
5 7
6 5
1 6
