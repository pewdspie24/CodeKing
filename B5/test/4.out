1000000000
11
1000000000
10077696
1000000000
6561
64
